techmatters/hrm
# CHI-2006: Move resources-search-complete -> job-complete

Needs to be rolled out along with this infrastructure config PR: https://github.com/techmatters/infrastructure-config/pull/245

## Description

* Move resources-search-complete -> job-complete
* Reconfigured lambddas to deploy shared job complete lambda to be deployed for both resource import complete and resource index complete queues
* Rejig error types a little

### Checklist
- N/A New tests added

### Related Issues
CHI-2006

### Verification steps

* Deploy both PRs (there will probably need to be a 'apply hrm -> deploy -> apply resources -> deploy' for deploying the job-complete lambda for the first time)
* Check for the new lambda being run in Datadog
